Model/ShimModel.php

```diff
@@ -54,6 +56,9 @@ public function __construct($id = false, $table = null, $ds = null) {
 			'alias' => $this->alias,
 			'schema' => $this->schemaName,
 		];
+		if (empty($this->actsAs) || (is_array($this->actsAs) && !in_array('Containable', $this->actsAs))) {
```

Comment:
Can containable also be a key?

'Containable' => [config array]

Reply:
Yes.